sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -760,16 +759,15 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the device twin object",
+            "description": "Returns the device twin object.",
             "schema": {
               "$ref": "#/definitions/Twin"
             }
           }
         }
       },
       "put": {
-        "summary": "Replaces tags and desired properties of a device twin.",
-        "description": "Replaces a device twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "Replaces the tags and desired properties of a device. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
```

Comment:
"Replaces a device twin" sounds better to me. Is this not updating the twin? The above statement feels like we are updating the device.

Reply:
I also left a comment below about PUT's in general. 
1) There is no Create or update
2) We should use consistently use "Replace" or "Update"

Reply:
It calls out that all we are updating here are tags and desired properties of a twin.

Reply:
I don't understand your second comment; do you mean using release/ update consistently across different APIs?

Reply:
yes. 2) is consistency across. Sometimes we say "replace", sometimes "update"

Reply:
For 1, are tags just not properties of the twin? Or do properties mean twin properties :) Sorry I am still confused about this word. It is overloaded :)

Reply:
And why is there no Create or Update on a PUT? Does PUT not create the object?

Reply:
The "device twin" includes reported properties, so I like Abhipsa's point here

Reply:
1. For replace vs update - they mean different things. Replace points to PUT operation, where the entire object is replaced. Update is the PATCH operation, where we are patching a part of the twin object.
So yes, we will have both "replace" and "update" in our APIs

2. This is how tags and properties are placed in a twin representation:

```
"tags": {
	"$etag": "123",
	"deploymentLocation": {
		"building": "43",
		"floor": "1"
	}
},
"properties": {
	"desired": {
		"telemetryConfig": {
			"sendFrequency": "5m"
		},
		"$metadata" : {...},
		"$version": 1
	},
	"reported": {
		"telemetryConfig": {
			"sendFrequency": "5m",
			"status": "success"
		},
		"batteryLevel": 55,
		"$metadata" : {...},
		"$version": 4
	}
}
```

Reply:
Here is my confusion. Does updating properties mean Twin.Properties or just any property on the Twin object like Twin.Capabilities?

Reply:
Updating properties from the service-end refers to updating the twin desired properties only. The update API, however, can update both desired properties and tags. 
This document here lists all the operations that you can do on a twin object, from the service-end: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-device-twins#back-end-operations

Reply:
This is the reason the API description calls out that all you can update are tags and desired properties.